packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -247,9 +248,33 @@ export class MessageWidget extends Component {
       ) {
         // Send notification of new message
         const avatars = props.avatar.get('items');
+        let cardsLength = 0;
+        let message = lastActivityFromThis.object.displayName;
```

Comment:
Since `lastActivityFromThis.object` is checked a few times in the added code, let's put an if block on this before entering.

```
        let message;
        if (lastActivityFromThis.object) {
```